wasp/apps/pager.py

```diff
@@ -82,19 +82,53 @@ class NotificationApp(PagerApp):
 
     def __init__(self):
         super().__init__('')
+        self.confirmation_view = wasp.widgets.ConfirmationView()
+        self.is_in_confirmation_view = False
 
     def foreground(self):
         notes = wasp.system.notifications
-
         id = next(iter(notes))
         note = notes[id]
         del notes[id]
         title = note['title'] if 'title' in note else 'Untitled'
         body = note['body'] if 'body' in note else ''
         self._msg = '{}\n\n{}'.format(title, body)
 
+        wasp.system.request_event(
+            wasp.EventMask.TOUCH
+            | wasp.EventMask.SWIPE_LEFTRIGHT
+        )
         super().foreground()
 
+    def swipe(self, event):
+        mute = wasp.watch.display.mute
+        if event[0] == wasp.EventType.UP:
+            if self._page >= self._numpages:
+                wasp.system.navigate(wasp.EventType.BACK)
+            self._page += 1
+        elif event[0] == wasp.EventType.LEFT:
+            self.confirmation_view.draw('clear notifications?')
+            self.is_in_confirmation_view = True
```

Comment:
I'd rather see this triggered with an additional swipe down (instead of the tactile pulse).